src/coalesce-vue-vuetify3/src/components/admin/c-admin-table-toolbar.vue

```diff
@@ -4,6 +4,7 @@
     class="c-admin-table-toolbar"
     density="comfortable"
     :color="color"
+    elevation="4"
```

Comment:
I can move this back - it just seemed like it was in a weird spot.

Reply:
Probably fine - direct usages of c-admin-table-toolbar are rare.